### ISSUE TITLE
[Bug]: Accessibility Issue with TabList Component

### ISSUE THREAD

**Issue by SamuelMorinFr** [microsoft/fluentui#27275]
### Library
Fluent UI React v9 (@fluentui/react-components)
### Issue
The `TabList` component violates the `aria-required-children` [accessibility rule](https://accessibilityinsights.io/info-examples/web/aria-required-children), because the parent element has a `tablist` role but it contains 2 `<i>` child elements, at first and last position, with `role=none`.
#### Target application
[Components / TabList - Default ⋅ Storybook](https://react.fluentui.dev/?path=/docs/components-tablist--default)
#### Element path
#storybook-preview-iframe;.fui-FluentProvider3 > div > .___1o1f0dt.f6jr5hl.f1i9oj3a > .___8n9zdb0.f1063pyq.fui-TabList
#### Snippet
`<div role="tablist" data-tabster="{&quot;mover&quot;:{&quot;cyclic&quot;:true,&quot;direction&quot;:2,&quot;memorizeCurrent&quot;:true}}" class="fui-TabList ___8n9zdb0 f22iagw f1063pyq fi64zpg flvyvdh f10pi13n f1q9h2pe">`
#### How to fix
    Fix any of the following:
      Element has children which are not allowed (see related nodes)
      Element has no aria-busy="true" attribute
#### Environment
Microsoft Edge version 111.0.1661.44
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

![image](https://user-images.githubusercontent.com/100727705/226653846-d32081a2-f009-4a09-8ac9-ed3998dae9af.png)

**Comment by smhigley** [comment]
This is a false positive, and has already been fixed upstream in axe-core. It'll stop flagging in Accessibility Insights when they pick up the version with the fix. The axe-core issue is linked from the other Fluent issue, so I'll close this one as well 👍
